src/commands/kv/get_key.rs

```diff
@@ -0,0 +1,41 @@
+// todo(gabbi): This file should use cloudflare-rs instead of our http::auth_client
+// when https://github.com/cloudflare/cloudflare-rs/issues/26 is handled (this is
+// because the GET key operation doesn't return json on success--just the raw
+// value).
+
+use cloudflare::framework::response::ApiFailure;
+
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::project::Project;
+
+pub fn get_key(
+    project: &Project,
+    user: &GlobalUser,
+    id: &str,
+    key: &str,
+) -> Result<(), failure::Error> {
+    let api_endpoint = format!(
+        "https://api.cloudflare.com/client/v4/accounts/{}/storage/kv/namespaces/{}/values/{}",
+        project.account_id, id, key
```

Comment:
consider url-encoding the key since it's part of the request url here

Reply:
Handled! :)